spec/features/user_views_map_spec.rb

```diff
@@ -89,6 +89,18 @@
     expect(page).to have_content t('map.messages.missingStudents')
   end
 
+  scenario 'missing students alert dismissed' do
```

Comment:
We've been wording these in a way such that the nesting of the contexts forms a complete sentence when all the descriptions are joined together (as they are in the RSpec output when a scenario fails). So:

```
feature 'User views map' do
  ...
  context 'with multiple bus positions' do
    ...
    scenario 'live-updated as locations change' do
      ...
```

...becomes the valid sentence *"User views map with multiple bus positions live-updated as locations change"*.

So this could be something like `scenario "with a closable warning when not all students are displayed"`.